pkg/services/control/types.proto

```diff
@@ -119,3 +119,33 @@ enum HealthStatus {
     // Storage node application is shutting down.
     SHUTTING_DOWN = 3;
 }
+
+// Shard description.
+message ShardInfo {
+    // ID of the shard.
+    bytes shard_ID = 1 [json_name = "shardID"];
+
+    // Path to shard's metabase.
+    string metabase_path = 2 [json_name = "metabasePath"];
+
+    // Path to shard's blobstore.
+    string blobstore_path = 3 [json_name = "Path"];
+
+    // Path to shard's write-cache, empty if disabled.
+    string writecache_path = 4 [json_name = "Path"];
```

Comment:
Same JSON names for blobstor and writecache.

Reply:
added prefix